src/exceptiongroup/_formatting.py

```diff
@@ -359,6 +359,46 @@ def format_exception_only(self):
     )
     sys.excepthook = exceptiongroup_excepthook
 
+# Ubuntu's system Python has a sitecustomize.py file that imports
+# apport_python_hook and replaces sys.excepthook.
+#
+# The custom hook captures the error for crash reporting, and then calls
+# sys.__excepthook__ to actually print the error.
+#
+# We don't mind it capturing the error for crash reporting, but we want to
+# take over printing the error. So we monkeypatch the apport_python_hook
+# module so that instead of calling sys.__excepthook__, it calls our custom
+# hook.
+#
+# More details: https://github.com/python-trio/trio/issues/1065
+if getattr(sys.excepthook, "__name__", None) in (
+    "apport_excepthook",
+    # on ubuntu 22.10 the hook was renamed to partial_apport_excepthook
+    "partial_apport_excepthook",
+):
+    # patch traceback like above
+    traceback.TracebackException.__init__ = (  # type: ignore[assignment]
+        PatchedTracebackException.__init__
+    )
+    traceback.TracebackException.format = (  # type: ignore[assignment]
+        PatchedTracebackException.format
+    )
+    traceback.TracebackException.format_exception_only = (  # type: ignore[assignment]
+        PatchedTracebackException.format_exception_only
+    )
+
+    from types import ModuleType
+
+    import apport_python_hook
+
+    assert sys.excepthook is apport_python_hook.apport_excepthook
+
+    # monkeypatch the sys module that apport has imported
+    fake_sys = ModuleType("trio_fake_sys")
```

Comment:
I would like this to be renamed, even though it doesn't matter that much.

Reply:
for sure, meant to do it but forgot.